scripts/kernel_monitor/kernel_monitor.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Linux Kernel LTS monitor script
+"""
+
+import urllib3
+import os
+import semver
+import smtplib
+import feedparser
+import logging
+
+from email.mime.text import MIMEText
+
+# Parent directory of this script
+script_path = os.path.dirname(os.path.realpath(__file__))
+
+# The folowing xml file is used to fetch the latest Linux kernel info
+kernel_xml_url = "https://www.kernel.org/feeds/kdist.xml"
+
+# distro_certs holds the path of the certificates bundle file provided
+# by the distro of your choice. This is needed to retrieve the xml file.
+# Fedora => /etc/ssl/certs/ca-bundle.crt
+# Ubuntu => /etc/ssl/certs/ca-certificates.crt
+distro_certs = "/etc/ssl/certs/ca-bundle.crt"
+
+# Log file path
+logfile="./example.log"
+
+# email config
+smtp_hostname='127.0.0.1'
+email_sender="no-reply@linuxlts.local"
+email_recipients="root@localhost"
+
+def get_current_lts():
+    lts_file=script_path + "/current_lts_kernel"
```

Comment:
I'd be tempted to put the current LTS version in `../versions.txt` so that all versions are kept in one place.